src/components/GamePanel.vue

```diff
@@ -50,16 +53,30 @@ const foundPairs = ref([] as CardPair[]);
 const canFlipCards = ref(true);
 const showModal = ref(false);
 const modalContent = ref({} as CardData);
+const hasPostError = ref();
+let memoryController: MemoryController;
 const isFinished = computed(
   () => foundPairs.value.length == cards.value.length / 2
 );
+
+watch(isFinished, (isFinished) => {
+  console.log("here");
+  if (isFinished) {
+    console.log("here1");
+    hasPostError.value = memoryController.postGameResult();
+    console.log("asdf" + hasPostError.value);
+  }
+  console.log("here2");
+});
```

Comment:
```suggestion
watch(isFinished, (isFinished) => {
  if (isFinished) {
    hasPostError.value = memoryController.postGameResult();
  }
});
```
Remove no helpful console logs.